src/bin/carbonadod.rs

```diff
@@ -11,11 +11,18 @@ use log::error;
 enum Commands {
     /// Start storage provider node with configured frontends
     Start,
+    /// Derive the parity npub from nsec
+    ParityNpub {
+        /// Nostr secret key
+        #[arg(long)]
+        nsec: String,
+    },
 }
 
 pub async fn try_main() -> Result<()> {
     match Commands::parse() {
         Commands::Start => carbonado_node::start().await?,
+        Commands::ParityNpub { nsec } => carbonado_node::parity_npub(&nsec)?,
```

Comment:
Can we refer to carbonado_node in this case as Self?

Commands::Start => Self::start().await?,
Commands::ParityNpub { nsec } => Self::parity_npub(&nsec)?,

Reply:
I guess that would be over-kill unless commands becomes exceedingly long